pkg/updater/resultstore/resultstore.go

```diff
@@ -70,6 +72,43 @@ type processedResult struct {
 	TargetResults   map[string][]*singleActionResult
 }
 
+// extractGroupID extracts grouping ID for a results based on the testgroup grouping configuration
+// Returns an empty string for no config or incorrect config
+func extractGroupID(tg *configpb.TestGroup, result *processedResult) string {
+	switch {
+	// P - build info
+	case tg.GetPrimaryGrouping() == configpb.TestGroup_PRIMARY_GROUPING_BUILD:
+		return identifyBuild(tg, result)
+	// F - date (YYYY-mm-dd)
+	case tg.GetFallbackGrouping() == configpb.TestGroup_FALLBACK_GROUPING_DATE:
```

Comment:
nit: imo we shouldn't include fallback_grouping at all in processing, we should just set a reasonable default for ResultStore. Users shouldn't need to set this usually, we should be able to figure it out by default based on the result source.

Reply:
do you suggest to use primary grouping and then if nothing set, resort to grouping by invocation ID?

Reply:
Yup. I don't think fallback_grouping will help for this result source; if we need it later we can introduce it again.

Reply:
I just removed fallback_grouping. Can you take a look again? @sultan-duisenbay @michelle192837